boards/arm/msp_exp432e401y_dk/msp_exp432e401y_dk_defconfig

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2021 pipei.sandor@gmail.com
+
+CONFIG_BOARD_MSP_EXP432E401Y_DK=y
+CONFIG_SOC_SERIES_MSP432E4XX=y
+CONFIG_SOC_MSP432E401Y=y
+CONFIG_CORTEX_M_SYSTICK=y
+
+# Floating point options
+CONFIG_FPU=y
```

Comment:
Any reason this is set on the _defconfig file? Are there any components directly making use of the FP?
Note that this by default enables FPU_SHARING as well.

Reply:
I have created the board default config file based on 
msp_exp432p401r_launchxl_defconfig where the CONFIG_FPU is set to y.
https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/msp_exp432p401r_launchxl/msp_exp432p401r_launchxl_defconfig

For the moment I don't have any board sample projects which needs this option.

Reply:
then probably remove it? It is not an option that should be present in the board definition. It should be enabled by applications that do FP calculations, or subsystems/drivers that do it as well.

Reply:
Ok. I will remove it.